scripts/aws/deploy.py

```diff
@@ -130,6 +130,8 @@ def update_stage(client, rest_api_id, stage_name, stage_settings):
                         help="Location of catalog IDA for request routing")
     parser.add_argument("--enterprise-host", required=False, default='',
                         help="Location of enterprise IDA for request routing")
+    parser.add_argument("--gateway-host", required=True,
```

Comment:
Please don't make this new field required. By setting `required=True`, you are introducing a breaking change. This change will immediately halt any deployment pipelines that rely on this script. Please also set the default to something innocuous like "localhost".